<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroEndereco</class>
 <widget class="QWidget" name="CadastroEndereco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="rotuloTituloFormulario">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cadastro de Endere√ßo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="caixaConteudo" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel{
font: 700 16pt &quot;Segoe UI&quot;;
color: red;

}

QLineEdit:disabled{
border:3px dashed black
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>200</number>
      </property>
      <item>
       <widget class="QWidget" name="linhaCep" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>200</number>
         </property>
         <item>
          <widget class="QLabel" name="rotuloCep">
           <property name="text">
            <string>CEP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="campoCep"/>
         </item>
         <item>
          <widget class="QPushButton" name="botaoBuscarCep">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="linhaLogradouro" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1,1">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QLabel" name="rotuloLogradouro">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Logradouro</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="campoLogradouro">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rotuloNumero">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Numero</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="campoNumero"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="linhaRegiao" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,1,3,1,1">
         <item>
          <widget class="QLabel" name="rotuloBairro">
           <property name="text">
            <string>Bairro</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="campoBairro"/>
         </item>
         <item>
          <widget class="QLabel" name="rotuloCidade">
           <property name="text">
            <string>Cidade</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="campoCidade"/>
         </item>
         <item>
          <widget class="QLabel" name="rotuloUF">
           <property name="text">
            <string>UF</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="campoUF"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
